app/src/main/java/com/animal_villa/MainViewModel.kt

```diff
@@ -1,4 +1,9 @@
 package com.animal_villa
+import com.animal_villa.dto.Animal
+import com.animal_villa.dto.Character
+import com.animal_villa.dto.Player
```

Comment:
Import the DTOs to access all the data classes and their attributes. This will help reduce technical debt and make the code usable. https://kotlinlang.org/docs/data-classes.html

Reply:
This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Keep the codebase as small as possible to do the job you want to do.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.